### PR TITLE
Issue #13345: Enable example tests for SingleLineJavadocCheckExamples…

### PR DESCRIPTION
**Pull request by prathm3**
Issue: #13345 

Enabled tests for SingleLineJavadocCheckExamplesTests.java

### CODE REVIEW

**Comment by romani** [comment]
@prathm3, , please execute `mvn verify` on your local before pushing to PR.
If you have problems with execution of this command, please let us know, we can help.

**Comment by romani** [comment]
We need this changes in commit https://dev.azure.com/romanivanovjr/romanivanovjr/_build/results?buildId=21400&view=logs&j=c902ebb4-c9f8-5f09-4e17-ff78fbbc842e&t=9ca98c81-ff64-58f0-9d03-a23ac1c4a111&l=1800

Just put in commit all that are changed on your local after mvn verify.

**Comment by prathm3** [comment]
@romani , thanks for the hint. Failed some checks with `Caused by: java.net.ConnectException: Network is unreachable (connect failed)`. I tried retriggering tests but same error is occurring. ( on local all tests are passing ). Do you think this issue is because of some changes ? ( I have seen some companies turn off some functionality ( jobs ) over weekends, is it same for check style ? )

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/singlelinejavadoc/Example1.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="SingleLineJavadoc"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.javadoc.singlelinejavadoc;
+// xdoc section -- start
+public class Example1 {
+  /** @see Math */ // violation
```
Comment (romani) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/singlelinejavadoc/Example1.java:11]: we have few possible messages from Check https://checkstyle.org/checks/javadoc/singlelinejavadoc.html#Violation_Messages
it is not clear which violation is here, please add message to this line `// violation 'some message'` message is regexp so you can provide only most meanigful part of original message

**Review by romani — CHANGES_REQUESTED** [review]
items:

**Comment by romani** [comment]
Moved to https://github.com/checkstyle/checkstyle/pull/14992

**Pull request closed without merging**